cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -26,17 +26,22 @@ const (
 
 	kubeDNSProbeSRV = "SRV"
 	kubeDNSProbeA   = "A"
+	coreDNSVersion  = "0.9.9"
 )
 
 // GetKubeDNSVersion returns the right kube-dns version for a specific k8s version
-func GetKubeDNSVersion(kubeVersion *version.Version) string {
-	// v1.8.0+ uses 1.14.5
-	// v1.9.0+ uses 1.14.6
-	// In the future when the kube-dns version is bumped at HEAD; add conditional logic to return the right versions
+func GetDNSVersion(kubeVersion *version.Version, dns string) string {
```

Comment:
I think it might be better to have `dns string` to validate against a list of constants and return an explicit error.

```go
const (
	CoreDNS = "CoreDNS"
	KubeDNS = "kube-dns"
)
```

And then something like this:
```go
switch dns {
case KubeDNS:
	// return the kube-dns version
case CoreDNS:
	// return the CoreDNS version
default:
	// return error
}
```

This would also mean adding an error return from this function, but I think that's worth the verbosity.